Doc/tools/touch-clean-files.py

```diff
@@ -6,16 +6,36 @@
 
 from pathlib import Path
 
-# Input file has blank line between entries to reduce merge conflicts
-with Path("Doc/tools/clean-files.txt").open() as clean_files:
-    CLEAN = [
+# Exclude these whether they're dirty or clean,
```

Comment:
Could consider adding an assert here, so that it's obvious why the script is failing if somebody forgets that they need to run it from the repo root:

```suggestion
assert Path("Doc").exists() and Path("Doc").is_dir(), "Must run this script from the repo root"

# Exclude these whether they're dirty or clean,
```